src/BuildingRegistry.Api.Extract/Infrastructure/Modules/MediatRModule.cs

```diff
@@ -22,7 +29,22 @@ protected override void Load(ContainerBuilder builder)
                 return type => ctx.Resolve(type);
             });
 
-            builder.RegisterAssemblyTypes(typeof(GetBuildingsHandler).GetTypeInfo().Assembly).AsImplementedInterfaces();
+            var useProjectionsV2ConfigValue = _configuration.GetSection("FeatureToggles")["UseProjectionsV2"];
```

Comment:
I don't like taking a dependency on configuration to have this toggle inside our MediatRModule